draft-packages/title-block-zen/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@kaizen/draft-zen-title-block",
+  "version": "1.4.5",
+  "description": "The draft Zen Title block component",
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf '*.d.ts' '*.js' '*.map' 'KaizenDraft/**/*.d.ts' 'KaizenDraft/**/*.js' 'KaizenDraft/**/*.map'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git",
+    "directory": "draft-packages/zen-title-block"
+  },
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-design-system/issues"
+  },
+  "files": [
+    "**/*",
+    "!**/*.ts",
+    "!**/*.tsx",
+    "**/*.d.ts",
+    "!**/*.spec.*",
+    "!**/*.snap",
+    "!draft-templates",
+    "!stories",
+    "!tsconfig.dist.json"
+  ],
+  "author": "",
+  "private": false,
+  "license": "MIT",
+  "dependencies": {
+    "@kaizen/component-library": "^7.17.2",
+    "@kaizen/draft-tag": "^1.4.5",
+    "@types/classnames": "^2.2.10",
+    "classnames": "^2.2.6",
+    "lodash": "^4.17.15",
+    "react-media": "^1.9.2"
```

Comment:
I can't see `react-media` used anywhere... Is it something you were going to add, but decided against?

Reply:
Same for `lodash`